Utilities/TestStreebog/StreebogPreprocess.c

```diff
@@ -0,0 +1,61 @@
+#include <UserFile.h>
+#include <Library/OcCryptoLib.h>
+#include <BigNumLib.h>
+
+CONST UINT8  TestMessage1[] = {
+  0x30, 0x31,
+  0x32, 0x33,
+  0x34, 0x35,
+  0x36, 0x37,
+  0x38, 0x39,
+  0x30, 0x31,
+  0x32, 0x33,
+  0x34, 0x35,
+  0x36, 0x37,
+  0x38, 0x39,
+  0x30, 0x31,
+  0x32, 0x33,
+  0x34, 0x35,
+  0x36, 0x37,
+  0x38, 0x39,
+  0x30, 0x31,
+  0x32, 0x33,
+  0x34, 0x35,
+  0x36, 0x37,
+  0x38, 0x39,
+  0x30, 0x31,
+  0x32, 0x33,
+  0x34, 0x35,
+  0x36, 0x37,
+  0x38, 0x39,
+  0x30, 0x31,
+  0x32, 0x33,
+  0x34, 0x35,
+  0x36, 0x37,
+  0x38, 0x39,
+  0x30, 0x31,
+  0x32
+};
+
+int
+ENTRY_POINT (
+  int   argc,
+  char  *argv[]
+  )
+{
+  UINT8  hash[64];
+
+  Streebog (TestMessage1, 63, hash, 256);
```

Comment:
This code only prints code and does not actually check the result against the control values.
Please include all the test values present in https://github.com/adegtyarev/streebog/tree/master/examples and make sure to compare them against the expected results. Print an error if this fails.